src/olympia/editors/models.py

```diff
@@ -29,8 +29,8 @@
 
 class CannedResponse(ModelBase):
 
-    name = TranslatedField()
-    response = TranslatedField(short=False)
+    name = models.CharField(max_length=255)
+    response = models.TextField()
```

Comment:
If there are no translations left in this model you need to remove the `pre_save` signal connect below the model declaration as well.